circleci/templates/gradle/config_template-v0_1.yml

```diff
@@ -16,18 +16,18 @@ jobs: #Defines the overall Jobs for this particular pipeline to be used in workf
             source $BASH_ENV
       - checkout
       - run:
-          name: Maven install without tests
+          name: Gradle install without tests
           command: |
-            ./gradlew install -B -DskipTests $DISABLE_DOWNLOAD_PROGRESS_OPTS
+            ./gradlew clean build --info
```

Comment:
Does this skip or execute the test phase?
The general idea with maven is that we skip the unit tests and defer them out to the next Unit Testing phase.